webui/react/src/App.tsx

```diff
@@ -149,7 +150,9 @@ const App: React.FC = () => {
     <HelmetProvider>
       <StoreProvider>
         <DndProvider backend={HTML5Backend}>
-          <AppView />
+          <SettingsProvider>
```

Comment:
Im not very worried about this so far cuz React18 can batch rendering, but if project gets bigger it might be a big rendering issue.